sdk/src/test/scala/com/horizen/integration/SidechainStateTest.scala

```diff
@@ -102,7 +102,7 @@ class SidechainStateTest
     transactionList.clear()
     transactionList += getRegularTransaction(1)
 
-    stateStorage.update(boxVersion, withdrawalEpochInfo, boxList.toSet, Set[Array[Byte]](), Set[WithdrawalRequestBox]())
+    stateStorage.update(boxVersion, withdrawalEpochInfo, boxList.toSet, Set[Array[Byte]](), Set[WithdrawalRequestBox](), false)
```

Comment:
Add tests with cases, when `containsBackwardTransferCertificate` is true.
The same for `SidechainStateStorage` tests

Reply:
still actual. Update SidechainState, SidechainStateStorage tests with the cases for logic you added.